hw15-myGsonWriter/src/test/java/ru/otus/l15/MyGsonWriterTest.java

```diff
@@ -0,0 +1,55 @@
+package ru.otus.l15;
+
+import jdk.jfr.Description;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.google.gson.Gson;
+
+import java.util.*;
+
+class MyGsonWriterTest {
+  Gson gson = new Gson();
+  private static BagOfPrimitivesAndArray getBagOfPrimitives(int a){
+    return new BagOfPrimitivesAndArray(a,"Hello",15,new int[]{1,2,3,4,5});
+  }
+
+  @Test
+  @Description("Object, containing primitives and array")
+  void getJson1() throws IllegalAccessException {
```

Comment:
Я сторонник того, чтобы  имена методов были говорящими, и Description тогда ни к чему.
```suggestion
  void primitivesAndArrayTest() throws IllegalAccessException {
```